tests/gtests/test_iface_attr_quantization.cpp

```diff
@@ -431,7 +431,12 @@ TEST_F(attr_quantization_test_t, TestLRN) {
     }
 }
 
+#define DEBUG_PRINT_LINE std::cout << "reached line " << __LINE__ << "\n";
```

Comment:
Is this needed?

Reply:
It was left over from debugging, removed it.